python/py-pyface/Portfile

```diff
@@ -5,7 +5,7 @@ PortSystem              1.0
 PortGroup               python 1.0
 PortGroup               github 1.0
 
-github.setup            enthought pyface 7.4.1
+github.setup            enthought pyface 8.0.0
```

Comment:
There should be a peg for legacy systems here. Depending on which Qt5 is minimally required, a threshold will vary; the portgroup will have thresholds listed. Otherwise it is safe to assume at least < 11 have no Qt5 (the last version to build on 10.6.8 x86_64 was an archaic 5.3) and must remain on Qt4-compatible one.